sdk/servicebus/service-bus/src/serviceBusAtomManagementClient.ts

```diff
@@ -212,6 +215,7 @@ export class ServiceBusManagementClient extends ServiceClient {
     let options: ServiceBusManagementClientOptions;
```

Comment:
Would also be great to allow passing in an prefix for the user agent string, so consumers of this library can set additional info (`<application_id>` in the guideline).

Reply:
Nice, I had seen it and thought of asking about the prefixes, good that you brought it up.

Reply:
Added UserAgentOptions for the prefix